console-subscriber/src/lib.rs

```diff
@@ -50,7 +51,12 @@ struct WakerVisitor {
     op: Option<WakeOp>,
 }
 
-struct Watch(mpsc::Sender<Result<proto::tasks::TaskUpdate, tonic::Status>>);
+struct Watch<T>(mpsc::Sender<Result<T, tonic::Status>>);
+
+enum WatchKind {
+    TaskUpdate(Watch<proto::tasks::TaskUpdate>),
+    TaskDetailUpdate(SpanId, Watch<proto::tasks::TaskDetails>),
```

Comment:
nit, not a blocker: i would personally have called these 
```suggestion
    Tasks(Watch<proto::tasks::TaskUpdate>),
    TaskDetails(SpanId, Watch<proto::tasks::TaskDetails>),
```
based on the name of the RPCs, rather than the names of the update message.

Reply:
Done in https://github.com/tokio-rs/console/pull/47/commits/d4d643f197df4c1c082dbb7665eeea5b516ac35a